docs/tutorials/getting-started.md

```diff
@@ -23,6 +23,18 @@ substituters = [...] https://hydra.iohk.io [...]
 
 This can be tricky to get setup properly. If you're still having trouble getting cache hits, consult the corresponding [troubleshooting section](../reference/troubleshooting#why-am-i-building-ghc).
 
+If you're running NixOS, you need to add/update the following in your `/etc/nixos/configuration.nix` files instead.
```

Comment:
Can you put this before the previous paragraph? I think that should be the last paragraph in this section.

Reply:
Good idea. I've pushed a commit